### DISCUSSION TITLE
p/5e398a2ca2d5462ebea28a3bf11784ba/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#44] [Announcements]
# p/5e398a2ca2d5462ebea28a3bf11784ba/

週報 2024-02-04今週毎日大体 7 時間寝ているので、昼には精神が良くなった。平日の午後にずっと外へ出かけないが、土曜日外へ遊びに出かけた。  散策木曜日と金曜日に気持ちが悪くいて、すると土曜日遊びに出かけた。学校から吉田神社で節分祭が開かれているから、そこに行った。そして、そこに北京からの Twitter で汁の人を暫く会った。 午前まずは吉田神社へ行ったり、そこに節分祭が開かれていたり

https://blog.rxliuli.com/p/5e398a2ca2d5462ebea28a3bf11784ba/

**Comment by iSharkChili** [comment]
hello 琉璃。

**Reply by rxliuli** [reply]
你好👋

**Reply by iSharkChili** [reply]
天呐，活捉一只技术宅🤪
